# message for posting messages in the gui
string data
uint8 color

uint8 COLOR_ORANGE = 0
uint8 COLOR_RED = 1
uint8 COLOR_GREEN = 2
uint8 COLOR_GRAY = 3
